site/content/posts/2021-06-7-Windows-Support.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Introducing Support for Windows Clusters
+image: /img/sonobuoy.svg
+excerpt: Sonobuoy can now run on Windows nodes meaning faster, consistent testing on all of your clusters.
+author_name: John Schnake
+author_url: https://github.com/vmware-tanzu
+categories: [kubernetes, sonobuoy, windows]
+# Tag should match author to drive author pages
+tags: ['Sonobuoy Team', 'John Schnake']
+date: 2021-06-07
+slug: sonobuoy-v0-51
```

Comment:
I think a better slug would be "sonobuoy-adds-windows-support"

Reply:
I wasn't sure what the slug was used for; just copied/edited from the v0.50 release blog. Is that for the URL?